admin_ui/backend/api/wizard.py

```
@@ -73,6 +73,108 @@ def _disk_preflight(path: str, *, required_bytes: int = 0) -> Tuple[bool, Option
+            results["messages"].append(msg)
+        results["success"] = True
+        return results
+    except Exception as e:
+        results["errors"].append(f"models permission fix failed: {e}")
+        return results
+
+
+def _ensure_models_dir_ready(path: str) -> None:
+    try:
+        os.makedirs(path, exist_ok=True)
+        return
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)